config/crd/bases/metallb.io_metallbs.yaml

```diff
@@ -43,6 +43,76 @@ spec:
           type: object
         status:
           description: MetallbStatus defines the observed state of Metallb
+          properties:
+            conditions:
+              items:
+                description: "Condition contains details for one aspect of the current
+                  state of this API Resource. --- This struct is intended for direct
+                  use as an array at the field path .status.conditions.  For example,
+                  type FooStatus struct{     // Represents the observations of a foo's
```

Comment:
Wrong indentation here?

Reply:
This file is generated (make manifests).